wcfsetup/install/files/lib/data/user/group/UserGroupAction.class.php

```diff
@@ -120,18 +120,25 @@ public function copy() {
 		
 		$optionValues = $statement->fetchMap('optionID', 'optionValue');
 		
-		$groupAction = new UserGroupAction([], 'create', [
+		$groupType = $this->groupEditor->groupType;
+		// When copying special user groups of which only one may exist,
+		// change the group type to 'other'.
+		if (in_array($groupType, [UserGroup::EVERYONE, UserGroup::GUESTS, UserGroup::USERS, UserGroup::OWNER])) {
+			$groupType = UserGroup::OTHER;
+		}
+		
+		/** @var UserGroup $group */
+		$group = (new UserGroupAction([], 'create', [
 			'data' => [
 				'groupName' => $this->groupEditor->groupName,
 				'groupDescription' => $this->groupEditor->groupDescription,
 				'priority' => $this->groupEditor->priority,
 				'userOnlineMarking' => $this->groupEditor->userOnlineMarking,
-				'showOnTeamPage' => $this->groupEditor->showOnTeamPage
+				'showOnTeamPage' => $this->groupEditor->showOnTeamPage,
+				'groupType' => $groupType,
```

Comment:
```suggestion
				// Prevent the duplication of built-in groups by forcing new groups to be of the "other" type.
				'groupType' => UserGroup::OTHER,
```

Reply:
I used my approach to keep the group type the same for moderated user groups (even if the leaders would are not copied).

Reply:
Good catch, I didn't thought about those. Okay, in this case, please disregard my comments.